package.json

```diff
@@ -11,6 +11,7 @@
     "karma": "cross-env NODE_ENV=test babel-node --plugins=transform-es2015-modules-commonjs ./node_modules/.bin/karma start",
     "babel": "babel src/ --out-dir dist/module",
     "babel-server": "babel ./server/*.js --ignore=node_modules --out-dir ./server",
+    "babel-server-test": "babel ./server/*.js --ignore=node_modules --out-dir ./demo/beaver-logger",
```

Comment:
Does it make sense to use `babel-node ./server/whatever` here to avoid transpiling to disk?

Reply:
@bluepnume So I need to get a new version of beaver-logger based on the changes and then run app.js in the demo folder.  I've not used `babel-node` before and after looking it up it seems it compiles and runs a file.  I see for the `webpack` script that it is compiling the `src` directory and storing in `dist`.  Not sure what to do here.  I am ignoring the generated files from being in the repo.

Reply:
All good. Your approach will totally work! 👍